Automattic/jetpack
# Modules: Add module info to the module-headings.php file

Fixes #8457

#### Changes proposed in this Pull Request:
* The module information should be added to the `module-headings.php` file during the build process. This will allow Jetpack to obtain the module information from this file rather than from the individual module files.

#### Jetpack product discussion
* n/a

#### Does this pull request change what data or activity we track or use?
* no

#### Testing instructions:
Testing this PR will require confirming that the modules are displayed and function properly. 
1. Install this branch. Activate and connect Jetpack.
2. In wp-admin, navigate to Jetpack -> Settings. Confirm that the modules settings are displayed and function properly.
3. While on a Jetpack dashboard page, scroll to the bottom of the page and click the Modules link. Verify that the modules are displayed and function properly.
4. Create a new site with this branch using JN. Verify that the default modules are activated.
